src/main/java/org/jabref/gui/mergeentries/newmergedialog/toolbar/ThreeWayMergeToolbar.java

```diff
@@ -176,6 +180,14 @@ public void setOnSelectRightEntryValuesButtonClicked(Runnable onClick) {
         selectRightEntryValuesButton.setOnMouseClicked(e -> onClick.run());
     }
 
+    public void saveGuiPreferences() {
+        preferencesService.getGuiPreferences().setMergePlainTextOrDiff(plainTextOrDiffComboBox.getValue());
```

Comment:
When I created the `PlainTextOrDiff` enum type, I thought it would be useful because I can embed the "Plain Text" and "Show Diff" texts in there, and display the labels in UI. In preferences, I think a boolean should suffice. Maybe a boolean property named `shouldShowDiffs`?

Reply:
This would have the advantage that you can then use bidrectional binding

Reply:
Implemented this for both PlainTextOrDiff and DiffView enums.